### ISSUE TITLE
got confused on understanding option 1

### ISSUE THREAD

**Issue by TIMESTICKING** [strawhatboy/clash-for-win-self-rules-appender#1]
1. I started the program as you mentioned.
2. I modified the .yml file of the clash, and got 
```
file modified: C:\Users\Lee\.config\clash\profiles\1652341540245.yml, could be an automatic update
Updating...
Successfully updated config file C:\Users\Lee\.config\clash\profiles\1652341540245.yml
```
which is great.
3. Then I updated the .yml file of the clash, and got exactly like what is above, and the configs I modified went missing and became the brand new .yml config file

Did I miss something, bc I really don't understand pretty much about option 1

**Comment by TIMESTICKING** [comment]
I think what the program does is when it detects the yml file changed, It injects the content of file `myconf.yml` into the new updated yml file, but what if I want to delete something

**Comment by TIMESTICKING** [comment]
and I also find that a module named `parser` in clash for windows does the same as this repo does

**Comment by strawhatboy** [comment]
> I think what the program does is when it detects the yml file changed, It injects the content of file `myconf.yml` into the new updated yml file, but what if I want to delete something

Yes, it just append your custom rules to the yml file. but deleting is not supported now 🤣



> and I also find that a module named `parser` in clash for windows does the same as this repo does

You're right, I used this script years ago when CFW has no such function. Now I'm using CFW with `global proxy` + `system proxy off` + proxy setting in other apps + `Switchy Omega` in browser. Only in few cases I will turn on the `system proxy`
